sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -755,11 +761,12 @@ def begin_analyze(  # type: ignore
         :type documents:
             list[str] or list[~azure.ai.textanalytics.TextDocumentInput] or
             list[dict[str, str]]
-        :param tasks: A list of tasks to include in the analysis.  Each task object encapsulates the parameters
-            used for the particular task type.
-        :type tasks: list[Union[~azure.ai.textanalytics.EntitiesRecognitionTask,
-            ~azure.ai.textanalytics.PiiEntitiesRecognitionTask, ~azure.ai.textanalytics.EntityLinkingTask,
-            ~azure.ai.textanalytics.KeyPhraseExtractionTask, ~azure.ai.textanalytics.SentimentAnalysisTask]]
+        :param actions: A heterogeneous list of actions to perform on the inputted documents.
+            Each task object encapsulates the parameters used for the particular task type.
+            The outputted action results will be in the same order you inputted your actions.
+            Can not put duplicate actions into list.
```

Comment:
maybe clarify that actions with the same parameters? so like actions need to be unique?

Reply:
ok, can do, thanks @maririos!